system/lib/gl/gl.c

```diff
@@ -1575,999 +1770,33 @@ void* emscripten_GetProcAddress(const char *name_) {
-  if (!strcmp(name, "glDeleteVertexArrays")) return emscripten_glDeleteVertexArrays;
-  if (!strcmp(name, "glGenVertexArrays")) return emscripten_glGenVertexArrays;
-  if (!strcmp(name, "glIsVertexArray")) return emscripten_glIsVertexArray;
-  if (!strcmp(name, "glGetUniformIndices")) return emscripten_glGetUniformIndices;
-  if (!strcmp(name, "glGetActiveUniformsiv")) return emscripten_glGetActiveUniformsiv;
-  if (!strcmp(name, "glGetActiveUniformName")) return emscripten_glGetActiveUniformName;
-  if (!strcmp(name, "glGetUniformBlockIndex")) return emscripten_glGetUniformBlockIndex;
-  if (!strcmp(name, "glGetActiveUniformBlockiv")) return emscripten_glGetActiveUniformBlockiv;
-  if (!strcmp(name, "glGetActiveUniformBlockName")) return emscripten_glGetActiveUniformBlockName;
-  if (!strcmp(name, "glUniformBlockBinding")) return emscripten_glUniformBlockBinding;
-  if (!strcmp(name, "glReleaseShaderCompiler")) return emscripten_glReleaseShaderCompiler;
-  if (!strcmp(name, "glShaderBinary")) return emscripten_glShaderBinary;
-  if (!strcmp(name, "glDepthRangef")) return emscripten_glDepthRangef;
-  if (!strcmp(name, "glClearDepthf")) return emscripten_glClearDepthf;
-  if (!strcmp(name, "glVertexAttribDivisor")) return emscripten_glVertexAttribDivisor;
-  if (!strcmp(name, "glGenFramebuffers")) return emscripten_glGenFramebuffers;
-  if (!strcmp(name, "glGenRenderbuffers")) return emscripten_glGenRenderbuffers;
-  if (!strcmp(name, "glBindFramebuffer")) return emscripten_glBindFramebuffer;
-  if (!strcmp(name, "glBindRenderbuffer")) return emscripten_glBindRenderbuffer;
-  if (!strcmp(name, "glGetRenderbufferParameteriv")) return emscripten_glGetRenderbufferParameteriv;
-  if (!strcmp(name, "glFramebufferRenderbuffer")) return emscripten_glFramebufferRenderbuffer;
-  if (!strcmp(name, "glRenderbufferStorage")) return emscripten_glRenderbufferStorage;
-  if (!strcmp(name, "glCheckFramebufferStatus")) return emscripten_glCheckFramebufferStatus;
-  if (!strcmp(name, "glDeleteFramebuffers")) return emscripten_glDeleteFramebuffers;
-  if (!strcmp(name, "glDeleteRenderbuffers")) return emscripten_glDeleteRenderbuffers;
-  if (!strcmp(name, "glFramebufferTexture1D")) return emscripten_glFramebufferTexture1D;
-  if (!strcmp(name, "glFramebufferTexture2D")) return emscripten_glFramebufferTexture2D;
-  if (!strcmp(name, "glFramebufferTexture3D")) return emscripten_glFramebufferTexture3D;
-  if (!strcmp(name, "glTexImage2D")) return emscripten_glTexImage2D;
-  if (!strcmp(name, "glGetShaderPrecisionFormat")) return emscripten_glGetShaderPrecisionFormat;
-
-  // If LEGACY_GL_EMULATION is on, allow access to those functions too
-#ifdef LEGACY_GL_EMULATION
-  if (!strcmp(name, "glDeleteObject")) return emscripten_glDeleteObjectARB;
-  if (!strcmp(name, "glGetObjectParameteriv")) return emscripten_glGetObjectParameterivARB;
-  if (!strcmp(name, "glGetInfoLog")) return emscripten_glGetInfoLogARB;
-  if (!strcmp(name, 
```

Comment:
why are these methods not in `emscripten_legacy_gl_emulation_GetProcAddress`?

Reply:
Yeah, can move them there.